clang/lib/DPCT/APINamesThrust.inc

```diff
@@ -1416,3 +1416,18 @@ CONDITIONAL_FACTORY_ENTRY(
 CALL_FACTORY_ENTRY("thrust::make_zip_iterator",  CALL("oneapi::dpl::make_zip_iterator",  ARG(0)))
 // thrust::make_reverse_iterator
 CALL_FACTORY_ENTRY("thrust::make_reverse_iterator",  CALL("oneapi::dpl::make_reverse_iterator",  ARG(0)))
+
+// thrust::exclusive_scan
+HEADER_INSERT_FACTORY(
+  HeaderType::HT_Numeric,
+  CONDITIONAL_FACTORY_ENTRY(
+    CheckArgCount(4),
+    CALL_FACTORY_ENTRY("thrust::exclusive_scan",
+      CALL("std::exclusive_scan",
+        makeMappedThrustPolicyEnum(0), ARG(1), ARG(2), ARG(3), ARG("0"))),
+    CALL_FACTORY_ENTRY("thrust::exclusive_scan",
+      CALL("std::exclusive_scan",
+        CALL("oneapi::dpl::execution::make_device_policy", QUEUESTR),
+        ARG(0), ARG(1), ARG(2), ARG("0")))
+  )
+)
```

Comment:
thrust::exclusive_scan() has six function template declarations, more detail pls refer to https://thrust.github.io/doc/group__prefixsums_ga0f1b7e1931f6ccd83c67c8cfde7c8144.html#ga0f1b7e1931f6ccd83c67c8cfde7c8144, your rewrite mapping code here should cover all these six function template declarations.

For example, thrust::copy_if() has four function template declarations, and Line1372~Line1413 covers all these four declarations.